packages/app/src/app/components/Preview/DevTools/index.tsx

```diff
@@ -52,7 +52,7 @@ export interface IViewAction {
 export interface IViewType {
   id: string;
   title: string | ((options: any) => string);
-  Content: React.ComponentClass<DevToolProps, any>;
+  Content: React.ComponentClass<DevToolProps, any> | React.SFC<DevToolProps>;
```

Comment:
@CompuIves This can just be `React.ComponentType` (see [@types/react/index.d.ts#L72](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/b639fa54459e6657bcd8e2f1b066ef7ee307d62b/types/react/index.d.ts#L72))

```suggestion
  Content: React.ComponentType<DevToolProps>;
```

> ```ts
> type ComponentType<P = {}> = ComponentClass<P> | FunctionComponent<P>;
> ```

Reply:
Awesome!! I was searching for this, thank you!